code-reviews/pull-requests.md

```diff
@@ -33,17 +33,31 @@ We should always aim to have pull requests be as small as possible, without losi
 1. They are easier to deploy; this is aligned with the strategy of release fast and release often.
 1. Minimizes possible conflicts and stale PRs, which are difficult to merge and keep in sync with master either because they're very dynamic or contain refactoring.
 
-However, we should avoid having PRs that include code that is without context or loosely coupled.
+However, we should avoid having PRs that include code that is without context or loosely coupled. There is no right size, but keep in mind that a code review is a collaborative process, and you should work with your reviewer to find out a reasonable size. Besides the size, remember that every PR should:
 
-<!-- TODO: Expand the topics below
+1. Be consistent and within context, 
+1. not break the build, and
+1. include related tests (UTs, integration, etc.) as part of the PR.
```

Comment:
Nit: To simplify we can remove the (UTs, integration, etc.) and still keep the same meaning

Reply:
Excellent feedback, simplified.